apis/v1/gateway_types.go

```diff
@@ -294,6 +294,28 @@ type GatewaySpec struct {
 	// +optional
 	// <gateway:experimental>
 	TLS *GatewayTLSConfig `json:"tls,omitempty"`
+
+	// DefaultScope, when set, configures the Gateway as a default Gateway,
```

Comment:
I think `DefaultScope` might be OK. I was also considering `DefaultGatewayAttachmentScope`. Not sure how consistent we are with "attachment" language all over the place with parentrefs and whatnot though. Consider this non-blocking, just a thought I had, but it's going to experimental so we'll have more time to do the hardest computer science problem: naming things.